_extras/e02-jekyll-lesson-example.md

```diff
@@ -1,17 +1,31 @@
 ---
-title: "Containers used in generating this lesson"
+title: "Using Docker with Jekyll - Containers Used in Generating this Lesson"
+layout: episode
 teaching: 20
 exercises: 0
 questions:
+- "What is an example of how I might use Docker instead of installing software?"
 - "How can containers be useful to me for building websites?"
 objectives:
+- "Use an existing container image and Docker in place of a complicated install."
```

Comment:
Very minor suggestion: the original context is likely sufficiently clear.
```suggestion
- "Use an existing container image and Docker in place of complicated software installation work."
```

Reply:
How do you feel about this version?
```suggestion
- "Use an existing container image and Docker in place of a complicated software installation."
```